docker-compose.yml

```diff
@@ -32,6 +32,7 @@ services:
       CONCOURSE_CLUSTER_NAME: dev
       CONCOURSE_ENABLE_PIPELINE_INSTANCES: "true"
       CONCOURSE_ENABLE_ACROSS_STEP: "true"
+      CONCOURSE_DISPLAY_USER_ID_PER_CONNECTOR: "local:name,ldap:user_id,github:name"
```

Comment:
Does this need to be committed? Would rather leave the `docker-compose.yml` tidy if not just to make sure we're not straying too far from end users' configuration.

Reply:
I intentionally added it, wanted to commit it. This configuration shows how to configure this option, and it fits to real end user's configuration, though github should change to use "username". Is it to commit this?

Reply:
Deleted.